core/station/impl/src/services/system.rs

```diff
@@ -734,39 +687,22 @@ mod install_canister_handlers {
     /// Starts the fund manager service setting it up to monitor the upgrader canister cycles and top it up if needed.
     pub fn monitor_upgrader_cycles(
```

Comment:
nit: this method is now doing more then monitoring the upgrader_cycles, maybe rename to `register_cycles_monitor` or similar